#50 Anonymisierung / Pseudonymisierung im ETL-Prozess

THGthub opened the pull request:
Anpassungen:

* Es werden keine Patientennamen (Vornamen, Nachname) mehr aus der Source-DB in die DWH-DB übernommen
* Geburts- und potentiell Sterbedatum werden aus der Source-DB nicht mehr in die DWH-DB übernommen
* Patienten-Ids werden pseudonymisiert

THGthub committed:
#50 Anonymisierung / Pseudonymisierung im ETL-Prozess

jessvasseur committed:
Revert "#50 Anonymisierung / Pseudonymisierung im ETL-Prozess"

This reverts commit 0002408cde72d6e3cbfdcb97386c043d0f5b6ef1.

jessvasseur reviewed (changes requested):
Skript läuft, aber Format durcheinander, dadurch sind Änderungen nicht mehr ersichtlich (komplettes Skript durch neues Skript ersetzt)

jessvasseur committed:
#50 Anonymisierung / Pseudonymisierung im ETL-Prozess

- erneuter Versuch aktualisiertes Notebook abzulegen (nach Revert des vorherigen Commits)

jessvasseur commented:
Der vorherige Commit wurde rückgängig gemacht (Ausgangssituation aus dem main Branch wiederhergestellt) und das gleiche Colab Notebook aus Google Drive nochmal hochgeladen. Hier funktioniert der Diff, sodass die Änderungen direkt ersichtlich sind.
